runtime/tests/automl_runtime/prophet/model_test.py

```diff
@@ -112,26 +114,30 @@ def test_validate_predict_cols(self):
         run_id = run.info.run_id
         prophet_model = mlflow.pyfunc.load_model(f"runs:/{run_id}/model")
 
-        with pytest.raises(MlflowException, match="Input data columns") as e:
+        with pytest.raises(MlflowException, match="Model is missing inputs") as e:
```

Comment:
Is this error changed because we log the model with schema and mlflow find that the input dataframe doesn't have the "ds" column?

Reply:
New error message is from mlflow